basics/processing-instructions/steel/api/src/sdk.rs

```diff
@@ -0,0 +1,49 @@
+use steel::*;
+
+#[repr(C)]
+#[derive(Clone, Copy, Debug, Pod, Zeroable, PartialEq)]
+pub struct GoToTheParkData {
```

Comment:
add to this to [`instruction.rs`](https://github.com/regolith-labs/steel?tab=readme-ov-file#file-structure) file, use [`!instruction()`](https://github.com/regolith-labs/steel?tab=readme-ov-file#instructions) macro

Reply:
Done! Thanks for the suggestion.